content/python/concepts/strings/terms/strip/strip.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: '.strip()'
+Description: 'Eliminates spaces at the beginning and end of a string, and specified characters when passed as an argument.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.strip()`** method removes any whitespace at the beginning and end of a string. It’s also possible to specify which characters will be removed; otherwise, all whitespaces will be removed.
```

Comment:
A couple of changes here. Let's also include a note further explaining when specified characters are removed.
```suggestion
The **`.strip()`** method removes any whitespace at the beginning and end of a string. Optionally, a string of characters can be passed in as an argument to remove those characters instead. Otherwise, all whitespaces will be removed.

> **Note**: The .strip() method removes specified characters only when they appear as the outermost leading and trailing characters in the string.
```